implants/lib/eldritch/src/runtime/messages/report_text.rs

```diff
@@ -0,0 +1,33 @@
+use super::{Dispatcher, Transport};
+use anyhow::Result;
+use pb::c2::{ReportTaskOutputRequest, TaskOutput};
+
+#[cfg_attr(debug_assertions, derive(Debug))]
+#[derive(Clone)]
+pub struct ReportTextMessage {
+    pub(crate) id: i64,
+    pub(crate) text: String,
+}
+
+impl ReportTextMessage {
+    pub fn text(&self) -> String {
+        self.text.clone()
+    }
+}
+
+impl Dispatcher for ReportTextMessage {
```

Comment:
So each field will need to be reported separately? Not a big deal but seems unusual.

Reply:
Not necessarily, we can aggregate these before dispatching.